app/models/guidance.rb

```diff
@@ -54,7 +54,7 @@ class Guidance < ActiveRecord::Base
   validates :published, inclusion: { message: INCLUSION_MESSAGE,
                                      in: BOOLEAN_VALUES}
 
-  validates :themes, presence: { message: PRESENCE_MESSAGE }
+  validates :themes, presence: { message: PRESENCE_MESSAGE }, if: :published?
```

Comment:
This might cause problems with the weird way we publish Guidance and Groups in the UI. We will have to make sure that UAT is thorough.